web/src/containers/activity/Overview.js

```diff
@@ -1,18 +1,19 @@
 import { FormLabel } from '@cmsgov/design-system-core';
 import PropTypes from 'prop-types';
-import React, { useMemo, useCallback } from 'react';
+import React, { useMemo, useCallback, useContext } from 'react';
 import { connect } from 'react-redux';
 
+import { ActivityContext } from './ActivityContext';
 import { t } from '../../i18n';
 import { updateActivity as updateActivityAction } from '../../actions/activities';
 import RichText from '../../components/RichText';
 import Instruction from '../../components/Instruction';
 import { Subsection } from '../../components/Section';
 import TextArea from '../../components/TextArea';
-import { selectActivityByKey } from '../../reducers/activities.selectors';
 
 const Description = props => {
-  const { activity, updateActivity } = props;
+  const { updateActivity } = props;
+  const { activity } = useContext(ActivityContext);
```

Comment:
`useContext` is a fancy-pants way to consume a context without using the `<ContextConsumer>` thing. The Overview component doesn't have tests right now, so I'm not sure how we'll test this. Maybe we'll just have to wrap it with an `ActivityContext.Provider`, I dunno.